package.json

```diff
@@ -54,6 +54,14 @@
   },
   "prettier": {},
   "resolutions": {
-    "**/@octokit/core": "4.2.0"
+    "**/@octokit/core": "4.2.0",
+    "glob-parent": "^5.1.2",
+    "node-fetch": "^2.6.7",
+    "minimist": "^1.2.8",
+    "mixme": "^0.5.2",
+    "minimatch": "^3.1.1",
+    "lodash": "^4.17.21",
```

Comment:
after your updates packages don't depend on this at all so this is redundant (likely the same story for `minimist`)